CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ install(EXPORT ArborXTargets
   DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/ArborX
 )
 
+set(ARBORX_VERSION "10799")
```

Comment:
I chose the simplest version. I understand that there's an opportunity to desync with `ARBORX_VERSION_STRING`, but I think it would be an overkill to introduce something heavier.

Reply:
> I would get rid of the string version.

That sounds reasonable to me.

> Also we should clarify with a comment how the version number is encoded.

Will do.